types/chance/index.d.ts

```diff
@@ -70,7 +71,7 @@ declare namespace Chance {
         color(opts?: Options): string;
         company(): string;
         domain(opts?: Options): string;
-        email(opts?: Options): string;
+        email(opts?: {domain: string}): string;
```

Comment:
technically, doing the following is accepted in chance.

```javascript
new Chance().email({})
```

it's another slight of the doc that it's not clearly conveyed, but this is allowed. the way this is typed wouldn't allow for that, right? it's a weird edge case, but should be supported.

```suggestion
        email(opts?: {domain?: string}): string;
```

Reply:
You know, now that I think about it a bit, the following is also technically allowed by chance.

```javascript
new Chance().email({a: 1})
```

obviously, that's a bad usage though so we shouldn't support that in the typing. so should we allow the empty object? the way you're typing the methods that have multiple options though, you are allowing the empty object for them. you're way prevents the weird empty object for methods with a _single_ param, which i'm starting to think is better. but my way makes it consistent across methods. not sure which is better. 🤔